README.md

```
@@ -1,3 +1,71 @@
+### 장기말 이동 규칙, UnitRule
+
+- [x] 각 장기말의 이동 규칙을 알고있다.
+
+### 장기판 내의 위치, Position
+
+- [x] 장기판 내의 위치를 관리한다.
+- [x] 범위를 벗어나서 위치를 생성하려하면 예외를 발생시킨다.
+
+### 출력 형식
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)